packages/countries/scripts/generateCountrySymbol.mjs

```diff
@@ -251,20 +291,42 @@ const generateCountrySymbolComponents = ({
       componentName: countryCodeToComponentName(countryCode),
       ariaLabel: countryName,
       viewBox: viewBox ?? "0 0 72 72",
+      borderRadius: "50%",
+      aspectRatioMultiplier: "1",
+    });
+    const sharpFileContents = Mustache.render(template, {
+      svgElements: sharpSvgPaths.data,
+      componentName: countryCodeToComponentName(countryCode) + "_Sharp",
+      ariaLabel: countryName,
+      viewBox: sharpViewBox ?? "0 0 72 50",
+      borderRadius: "0",
+      aspectRatioMultiplier: "1.44",
```

Comment:
Is there a way to make this a class rather than it being added like this?

Reply:
Yeah, could go the class route if you think that's better?

Was just following the precedent of `packages/countries/src/country-symbol/CountrySymbol.tsx` (which sets the `--saltCountrySymbol-size-multiplier` in the same manner)

Reply:
Made that change 👍 Moved these styles to the stylesheet keyed by a `saltSharpCountrySymbol` class

Reply:
let's chat about this name, I I'm not sure if it should be saltCountrySymbol-sharp instead to fit with every other class

Reply:
Makes sense, and consistency always gets my vote 👍